vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/chromosomewsclient/ChromosomesWSOutput.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.vcfdump.chromosomewsclient;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class ChromosomesWSOutput {
+
+    private Response[] response;
+
+    public ChromosomesWSOutput() {
+    }
+
+    public Response[] getResponses() {
+        return response;
+    }
+
+    public void setResponse(Response[] responses) {
+        this.response = responses;
+    }
+
+    Set<String> getAllChromosomeNames() {
```

Comment:
No real need for `all` in the name.

Also why make this method package protected and not the other getter/setter?